src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -39,8 +41,43 @@ However, instructions are provided for the steps you need to take should you cho
 
 {% capture content_with_toc %}
 
+## Testing instructions for Firefox for Android 79+ (Fenix)
+:::note alert
+Persistent loading is currently not supported. These instructions are for temporary loading.
+:::
+
+- Install or update [web-ext](https://github.com/mozilla/web-ext) to version 4.1.0 or later.
+
+- Install or update to the latest version of [Firefox Nightly for Android](https://play.google.com/store/apps/details?id=org.mozilla.fenix).
+
+- Make sure you have adb installed ([Linux](https://dl.google.com/android/repository/platform-tools-latest-linux.zip), [Mac](https://dl.google.com/android/repository/platform-tools-latest-darwin.zip), [Windows](https://dl.google.com/android/repository/platform-tools-latest-windows.zip)) and in your PATH.
+
+- Connect your Android device via USB.
+
+- Make sure you have [Android's USB debugging](https://developer.android.com/studio/debug/dev-options) enabled. You can run `adb devices` to verify.
+
+- In the settings for Firefox Nightly, enable "Remote debugging via USB."
+
+- In the unzipped directory of your extension, run `web-ext run -t firefox-android` and follow the instructions on screen to make sure you select the right device. Select `org.mozilla.fenix` as a target.
+
+{% endcapture %}
+{% include modules/column-w-toc.liquid
+  id: "testing-instructions-fenix"
+  content: content_with_toc
+%}
+
+<!-- END: Content with Table of Contents Module -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
 ## Set up your computer and Android emulator or device
 
+::: note alert
+This section only applies to Firefox for Android < 68 (Fennec)
+:::
+
```

Comment:
The "on your development computer" part of this section does look actually not strictly related to Fennec (it is basically how to install adb), maybe we should split out (or just call out) the parts that are related to Fennec (which seems to be basically all the content in this subsection starting from "On you device or android emulator").